### PR TITLE
enh(script): add missing spaces

### PR DESCRIPTION
**Pull request by sc979**
## Description

**Do not merge until the 20.10.0 is released**

Add spaces after each script's queries

before : 
```
Are you using a proxy ? (y/n)n
...
Which IP do you want to use as CURRENT NODE IP ?1
```

after

```
Are you using a proxy ? (y/n) n
...
Which IP do you want to use as CURRENT NODE IP ? 1

```
**Fixes** # (none)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [x] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.04.x
- [ ] 19.10.x
- [ ] 20.04.x
- [x] 20.10.x (master)

### CODE REVIEW

**Comment by sc979** [comment]
**Do not merge until the 20.10.0 is released**

**Review by kduret — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]

**Comment by Nohzoh** [comment]
About spaces, you should avoid spaces before the ponctuation in english.
https://simple.wikipedia.org/wiki/Question_mark

> In English, (...), using this space is thought of as bad form.

File: bin/registerServerTopology.php [hunk 1]
```diff
@@ -137,7 +137,7 @@
         $configOptions['TARGET_NODE_ADDRESS'] . ': please enter your password ',
         true
     );
-    $configOptions['PROXY_USAGE'] =  strtolower(askQuestion("Are you using a proxy ? (y/n)"));
+    $configOptions['PROXY_USAGE'] =  strtolower(askQuestion("Are you using a proxy ? (y/n) "));
```
Comment (Nohzoh) [inline bin/registerServerTopology.php:140]: In English we shouldn't use space before ponctuation
```suggestion
    $configOptions['PROXY_USAGE'] =  strtolower(askQuestion("Are you using a proxy? (y/n) "));
```

File: bin/registerServerTopology.php [hunk 2]
```diff
@@ -198,7 +198,7 @@
 
 while (!$goodIp) {
     echo $ipSelection;
-    $ipChoice = askQuestion('Which IP do you want to use as CURRENT NODE IP ?');
+    $ipChoice = askQuestion('Which IP do you want to use as CURRENT NODE IP ? ');
```
Comment (Nohzoh) [inline bin/registerServerTopology.php:201]: In English we shouldn't use space before ponctuation
```suggestion
    $ipChoice = askQuestion('Which IP do you want to use as CURRENT NODE IP? ');
```

File: bin/registerServerTopology.php [hunk 3]
```diff
@@ -246,7 +246,7 @@
 
 echo $summary;
 
-$proceed = askQuestion('Do you want to register this server with those information ? (y/n)');
+$proceed = askQuestion('Do you want to register this server with those information ? (y/n) ');
```
Comment (Nohzoh) [inline bin/registerServerTopology.php:249]: In English we shouldn't use space before ponctuation
```suggestion
$proceed = askQuestion('Do you want to register this server with those information? (y/n) ');
```

**Review by Nohzoh — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]

**Pull request merged**